From d7c8d0050167f60124ed1ec017fecd74caeddd6f Mon Sep 17 00:00:00 2001
From: zachary <zhangzg@marvell.com>
Date: Wed, 28 Dec 2016 23:52:09 +0800
Subject: [PATCH 1239/2241] spi: a3700: set spi max output clock to 100MHz

- Previously, max SPI output clock to devices was set
  to 50MHz.
- This patch increases it to 100MHz, which is the real
  max output clock of the SPI controller. And it would
  improve spi flash read/write performance.

Change-Id: Icde645d4e2f2e043ffd63c4c9988530c02781bee
Signed-off-by: zachary <zhangzg@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/35099
Reviewed-by: Hua Jing <jinghua@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
---
 drivers/spi/spi-armada-3700.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/spi/spi-armada-3700.c b/drivers/spi/spi-armada-3700.c
index dbf0a70..d1918260 100644
--- a/drivers/spi/spi-armada-3700.c
+++ b/drivers/spi/spi-armada-3700.c
@@ -100,7 +100,7 @@
 
 #define DATA_CAP_CLOCK_CYCLE_NS		14	/*Minimum clock cycle of flash*/
 #define RVT_EDGE_FLASH_CAP_FREQ		(1000 * 1000 * 1000 / DATA_CAP_CLOCK_CYCLE_NS)
-#define A3700_SPI_MAX_OUTPUT_CLK_FREQ		(50 * 1000 * 1000)	/*50MHz*/
+#define A3700_SPI_MAX_OUTPUT_CLK_FREQ		(100 * 1000 * 1000)	/*100MHz*/
 
 struct a3700_spi_initdata {
 	unsigned int cs_num;
-- 
2.7.4

